### ISSUE TITLE
Stylelint ignore stylelint-disable command

### ISSUE THREAD

**Issue by sibicode** [stylelint/stylelint#3453]
> Clearly describe the bug

Stylelint ignore **stylelint-disable** command 

> Which rule, if any, is the bug related to?

`any`

> What CSS is needed to reproduce the bug?

`index.css`
```css
@import "foo.css"

/* stylelint-disable */
@import "../../node_modules/slick-carousel/slick/slick.css";
/* stylelint-enable */

@import "bar.css"
```

> What stylelint configuration is needed to reproduce the bug?

```json
{
  "extends": "stylelint-config-standard",
  "rules": {
    "string-quotes": "single"
  }
}
```

> Which version of stylelint are you using?

 `9.3.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

`"gulp-postcss": "^7.0.1", and" postcss-import": "^11.1.0"`
```js
require('postcss-import')({
      plugins: [
        require('stylelint')({
          reporters: [
            {formatter: 'string', console: true}
          ]
        })
      ]
    })
```
`gulp task`
```js
gulp.task('styles', () => (
  gulp.src('src/css/index.css')
    .pipe(postcss(PLUGINS))
    .pipe(gulp.dest('build/css/'))
))
```

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

Not, pure css

> What did you expect to happen?

"No warnings to be flagged."

> What actually happened (e.g. what warnings or errors did you get)?

"The following warnings were flagged:"

```shell
gulp-postcss: index.css
stylelint: /node_modules/slick-carousel/slick/slick.css:6:5: Expected indentation of 2 spaces (indentation)
stylelint: /node_modules/slick-carousel/slick/slick.css:7:5: Expected indentation of 2 spaces (indentation)
stylelint: /node_modules/slick-carousel/slick/slick.css:9:5: Expected indentation of 2 spaces (indentation)
stylelint: /node_modules/slick-carousel/slick/slick.css:10:8: Expected indentation of 2 spaces (indentation)
... etc
```

**Comment by hudochenkov** [comment]
Thank you for using a template.

Could you provide full warnings please, because what did you provide doesn't show anything related to stylelint.

**Comment by sibicode** [comment]
Here are these errors:

```js
stylelint: /node_modules/slick-carousel/slick/slick.css:6:5: Expected indentation of 2 spaces (indentation)
stylelint: /node_modules/slick-carousel/slick/slick.css:7:5: Expected indentation of 2 spaces (indentation)
stylelint: /node_modules/slick-carousel/slick/slick.css:9:5: Expected indentation of 2 spaces (indentation)
stylelint: /node_modules/slick-carousel/slick/slick.css:10:8: Expected indentation of 2 spaces (indentation)
....
```

But styleline should ignor this file..

**Comment by jeddy3** [comment]
_Wrapping_ an `@import` statement in stylelint disable commands will not work as you're using the `plugins` option of postcss-imports i.e. stylelint is [run on each file](https://github.com/postcss/postcss-import#plugins) that the `postcss-import` plugin imports.

Of the three options in https://github.com/stylelint/stylelint/issues/3452#issuecomment-403730135, I think that either putting stylelint as the _first plugin in your PostCSS pipeline_ or using [stylelint-webpack-plugin](https://github.com/webpack-contrib/stylelint-webpack-plugin) will work for you.

Alternatively, you can use stylelint as a standalone tool on the CLI.
